### DISCUSSION TITLE
How to disable dark mode?

### DISCUSSION THREAD

**Discussion by vkaelin** [slidevjs/slidev#1087] [Q&A] [answered]
Hello!

How can I only let the user use the light theme of the presentation?

I didn't find this option in the doc, only when exporting it to a PDF but I'm speaking about the web app itself.

Thanks a lot!

**Comment by antfu** [comment, marked as answer]
https://sli.dev/custom/#frontmatter-configures

```md
---
colorSchema: light
---
```

**Reply by vkaelin** [reply]
Oh I missed it completely, thanks a lot!

**Reply by ScDor** [reply]
Thanks! Any way to control the top level title coloring to make it more visible? 
![image](https://github.com/user-attachments/assets/829044c9-51c6-42b2-b8e8-adfab0f57ae6)
